package/Dockerfile

```diff
@@ -233,8 +233,8 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.9.3-rc1
-ENV CATTLE_DASHBOARD_UI_VERSION v2.9.3-rc1
+ENV CATTLE_UI_VERSION 2.9.3
+ENV CATTLE_DASHBOARD_UI_VERSION v2.9.3
```

Comment:
I'm a bit confused with this. Looking at v2.9.3:
https://github.com/rancher/rancher/blob/v2.9.3/package/Dockerfile#L236-L237

It's also in hotfix branch: https://github.com/rancher/rancher/blob/v2.9.3-hotfix-schema-leak/package/Dockerfile#L236-L237

So why is github showing this here :thinking:

Reply:
Oh i didn't notice this, yeah I'm not sure either... I don't think it showed this when creating the PR either